packages/EasyPagination/src/Paginators/DoctrineDbalLengthAwarePaginator.php

```diff
@@ -64,7 +64,13 @@ protected function doGetResult($queryBuilder): array
      */
     protected function doGetTotalItems($queryBuilder, string $countAlias): int
     {
-        $result = (array)$this->conn->fetchAssoc($queryBuilder->getSQL(), $queryBuilder->getParameters());
+        // @TODO remove this method_exists check when doctrine/dbal is updated to a version containing the fetchAssociative method (>=2.11)
+        $methodName = \method_exists($this->conn, 'fetchAssociative') ? 'fetchAssociative' : 'fetchAssoc';
+        $result = (array) $this->conn->{$methodName}(
+            $queryBuilder->getSQL(),
+            $queryBuilder->getParameters(),
+            $queryBuilder->getParameterTypes()
```

Comment:
Please fix the tests failing in CI (there is no expectation for the `getParameterTypes` method call there).